client/blocks/importer/wordpress/upgrade-plan/constants.tsx

```diff
@@ -40,24 +45,56 @@ export const UpgradePlanHostingTestimonials = [
 	},
 ];
 
+export const defaultHostingDetails = [
+	{
+		title: translate( 'Reduced error rate' ),
+		description: translate( '16% fewer errors' ),
+		icon: shield,
+	},
+	{
+		title: translate( 'Increased speed' ),
+		description: translate( '30% faster' ),
+		icon: trendingUp,
+	},
+	{
+		title: translate( 'Higher availability' ),
+		description: translate( '3% better uptime' ),
+		icon: chartBar,
+	},
+] as Array< { title: string; description: string | ReactNode; icon: JSX.Element } >;
+
 export function useUpgradePlanHostingDetailsList() {
 	const translate = useTranslate();
+	const { siteMetricData, showUpdatedSpeedMetrics } = UseGetUpgradePlanSiteMetrics();
+	const hostingDetails = [ ...defaultHostingDetails ];
 
-	return [
-		{
-			title: translate( 'Reduced error rate' ),
-			description: translate( '16% fewer errors' ),
-			icon: shield,
-		},
-		{
-			title: translate( 'Increased speed' ),
-			description: translate( '30% faster' ),
+	if ( showUpdatedSpeedMetrics ) {
+		const wordpressLCP = Math.round( 100 * wordpressCwvtechReportJson?.goodLCP );
+		const percentageDifference =
+			siteMetricData?.basic?.lcp &&
+			Math.round(
+				100 *
+					Math.abs(
+						( siteMetricData?.basic?.lcp - upgradePlanSiteMetricsLcpThreshold ) /
+							( ( siteMetricData?.basic?.lcp + upgradePlanSiteMetricsLcpThreshold ) / 2 )
+					)
+			);
+		const updatedHostingSpeedDetails = {
+			title: translate( 'Higher speed' ),
```

Comment:
Should we ask for translations already?

Reply:
If we're sure about the text for the time being then yes we should start asking for translations. We can always deploy for english locale first if necessary.